src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -185,6 +186,26 @@ protected final DefaultConfiguration createTreeWalkerConfig(Configuration config
         return dc;
     }
 
+    /**
+     * Creates {@link DefaultConfiguration} or the {@link Checker}.
+     * based on the the list of {@link Configuration}.
+     *
+     * @param configs list of {@link Configuration} instances.
+     * @return {@link DefaultConfiguration} for the {@link Checker}.
+     */
+    protected final DefaultConfiguration createTreeWalkerConfig(List<Configuration> configs) {
+        final DefaultConfiguration dc =
+                new DefaultConfiguration("configuration");
```

Comment:
use variable for this `static final private CUSTOM_MODULE = "configuration"`

Reply:
Done

Reply:
not done, use CUSTOM_MODULE here